src/constants/extensions.ts

```diff
@@ -130,6 +130,8 @@ export const supportedExtensionsConfig: ExtensionConfig[] = [
     descriptionShort: MSG.oneTxPaymentDescriptionShort,
     descriptionLong: MSG.oneTxPaymentDescriptionLong,
     neededColonyPermissions: [ColonyRole.Administration, ColonyRole.Funding],
+    currentVersion: 3,
```

Comment:
Rather than hardcoding the current version, there's an `availableVersion` field on `AnyExtensionData` which indicates the highest available version stored in Amplify